### ISSUE TITLE
ZoomImage not showing

### ISSUE THREAD

**Issue by joostmeijles** [laurenashpole/react-inner-image-zoom#48]
The ZoomImage is not shown when the `onLoad` event is triggered before the ZoomImage element is visible / rendered in the DOM.

See the movie below:
![onload-render-error](https://user-images.githubusercontent.com/4342282/116896812-e11d4200-ac34-11eb-913f-4aabcdeb7165.gif)

I was only able to reproduce this behaviour on mobile not on desktop.

**Comment by laurenashpole** [comment]
I've tried looking into this but I'm having trouble reproducing. I got it to happen once on the demo site (before I'd opened my console of course) but can't get it to happen locally even pointing to the same Github hosted images.

It shouldn't be possible for the onLoad event to be triggered before isZoomed is set to true and isZoomed also handles showing and hiding the close button so that's a bit of a mystery. It may have to do with resetting the image when opening and closing the fullscreen modal so I'll keep investigating down that path.
